src/docs/product/discover-queries/query-builder.mdx

```diff
@@ -46,6 +46,8 @@ The Query Builder syntax is identical to [Sentry's Search syntax](/product/sentr
 
 Use `OR` and `AND` search conditions between filters. However `OR` cannot be used between aggregate and non-aggregate filters. For more details about these conditions, see [Using `OR` and `AND`](/product/sentry-basics/search/#using-or-and-and).
 
+You can also search multiple values on the same key without an operator, and the search terms automatically use an `OR` operator. For example, "x:value1 x:value2" is the same as "x:value1 `OR` x:value2". For more details, go to [Multiple Values on the Same Key](/product/sentry-basics/search/#multiple-values-on-the-same-key).
```

Comment:
Hmm might be missing context here, but I think this is incorrect 
`x:value1 x:value2` is still equivalent to `x:value1 AND x:value2` so i think this should read:
```suggestion
You can also search multiple values on the same key without an operator, and the search terms automatically use an `OR` operator. For example, "x:[value1, value2]" is the same as "x:value1 `OR` x:value2". For more details, go to [Multiple Values on the Same Key](/product/sentry-basics/search/#multiple-values-on-the-same-key).
```

Reply:
Yeah, `x:value1 x:value2` is equivalent to `x:value1 and x:value2`, and so will return no results.